src/app/frontend/poddetail/creatorinfo.html

```diff
@@ -16,52 +16,114 @@
-
-          <div class="kd-zerostate-message">
-            <kd-zerostate>
-              <div class="kd-zerostate-title">[[There is nothing to display here|Title for pods card zerostate in pod details page.]]</div>
-              <div class="kd-zerostate-text">[[This Pod does not have a creator.|Text for pod creator zerostate in pod details page.]]</div>
-            </kd-zerostate>
+    <kd-resource-card-list selectable="false"
+                           with-statuses="true">
+      <kd-resource-card-list-header>
+        [[Created by|Heading for Creator card on pod detail page.]]
+      </kd-resource-card-list-header>
+      <kd-resource-card-header-columns>
+        <kd-resource-card-header-column size="small"
+                                        grow="2">
+          [[Name|Label 'Name' which appears as a column label in the creator card.]]
+        </kd-resource-card-header-column>
+        <kd-resource-card-header-column size="small"
+                                        grow="1">
+          [[Kind|Label 'Kind' which appears as a column label in the creator card.]]
+        </kd-resource-card-header-column>
+        <kd-resource-card-header-column size="small"
+                                        grow="2"
+                                        ng-if="$ctrl.areMultipleNamespacesSelected()">
+          [[Namespace|Label 'Namespace' which appears as a column label in the creator card.]]
+        </kd-resource-card-header-column>
+        <kd-resource-card-header-column size="small"
+                                        grow="2">
+          [[Labels|Label 'Labels' which appears as a column label in the creator card.]]
+        </kd-resource-card-header-column>
+        <kd-resource-card-header-column size="small"
+                                        grow="1">
+          [[Pods|Label 'Pods' which appears as a column label in the creator card.]]
+        </kd-resource-card-header-column>
+        <kd-resource-card-header-column size="small"
+                                        grow="1">
+          [[Age|Label 'Age' which appears as a column label in the creator card.]]
+        </kd-resource-card-header-column>
+        <kd-resource-card-header-column size="small"
+                                        grow="2">
+          [[Images|Label 'Images' which appears as a column label in the creator card.]]
+        </kd-resource-card-header-column>
+        <kd-resource-card-header-column size="small"
+                                        grow="nogrow">
+        </kd-resource-card-header-column>
+      </kd-resource-card-header-columns>
+      <kd-resource-card object-meta="$ctrl.creator.objectMeta"
+                        type-meta="$ctrl.creator.typeMeta">
+        <kd-resource-card-status layout="row">
+          <md-icon class="material-icons md-warn"
+                   ng-if="::$ctrl.hasWarnings()">
+            error
+            <md-tooltip>One or more pods have errors</md-tooltip>
```

Comment:
Yes, if one of the pods has some errors then controller status will be displayed as red error icon just like below:

![image](https://cloud.githubusercontent.com/assets/2823399/24298363/14938358-10a6-11e7-8ead-b0f96c837b32.png)